docs/readme.md

```diff
@@ -89,6 +85,14 @@ This will generate the `site` directory which will contain all the gererated doc
 5. Go to your individual virtualization-sdk repo's settings, scroll to the bottom and verify under the GitHub Pages section the `Source` is set to `gh-pages branch`.
 6. Right above this will be a link explaining where your docs are published.
 
+You can also utilize the GitHub workflow for publishing docs (`.github/workflows/publish-docs.yml`). Make sure that the
+workflow is on a branch called `docs/x.y.z`. If you have more than one `docs/x.y.z` branch, you have to push your doc
```

Comment:
could you clarify what this means?
> Make sure that the workflow is on a branch called `docs/x.y.z`

Do we create a branch in our local fork? Is there something else that we would have to do to add the GitHub action for our fork?

Reply:
I rewrote this paragraph.